server/src/main/java/org/opensearch/common/settings/FeatureFlagSettings.java

```diff
@@ -34,7 +34,7 @@ protected FeatureFlagSettings(
         FeatureFlags.IDENTITY_SETTING,
         FeatureFlags.TELEMETRY_SETTING,
         FeatureFlags.DATETIME_FORMATTER_CACHING_SETTING,
-        FeatureFlags.WRITEABLE_REMOTE_INDEX_SETTING,
+        FeatureFlags.TIERED_WRITEABLE_REMOTE_INDEX_SETTING,
```

Comment:
Can we use `TIERED_REMOTE_STORE_INDEX_SETTING` ? This covers both tiering and remote store index aspect. By default, remote store backed index supports read/write so we don't have to really mention that in feature flag.

Reply:
In that case, I think `TIERED_REMOTE_INDEX_SETTING` should be sufficient. STORE also is redundant, let me know if that sounds good to u as well

Reply:
I am fine with remote Index as well. To me remote store index was more explicit in terms of index store is remote vs index is remote (which is not the case here).